internal/model/requestquery.go

```diff
@@ -0,0 +1,55 @@
+package model
+
+import (
+	"errors"
+	"gorm.io/gorm"
+)
+
+type Status string
+type Type string
+
+const (
+	StatusOngoing   Status = "ongoing"
+	StatusAutoreply Status = "autoreply"
+	StatusPending   Status = "pending"
+	StatusClosed    Status = "closed"
+	StatusReviewed  Status = "reviewed"
+)
+
+const (
+	TypeUnknown Type = "unknown"
+	TypeQuery   Type = "query"
+	TypeRequest Type = "request"
+)
+
+type RequestQuery struct {
+	gorm.Model
+	Status    Status
+	Type      Type
+	BookingId *uint
+	Booking   *Booking
```

Comment:
same w this, is this a 'belongs-to' booking? will it be an issue if theres no booking tied to the requestquery? (ignore if its my skill issue)

Reply:
see above